__tests__/article.test.js

```diff
@@ -151,5 +151,131 @@ describe("05 GET /api/articles", () => {
+
+      const output = {
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: new Date("2020-07-09T20:11:00.000Z"),
+        votes: 101,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(body)
+        .expect(200)
+        .then(() => {
+          return db.query("SELECT * FROM articles WHERE article_id = 1");
+        })
+        .then(({ rows }) => {
+          expect(rows[0]).toEqual(output);
+        });
+    });
+
+    test("The call should also function if the change is to remove votes", () => {
+      const body = { inc_votes: -1 };
+
+      const output = {
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: "2020-07-09T20:11:00.000Z",
+        votes: 99,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(body)
+        .expect(200)
+        .then(({ body }) => {
+          const { article } = body;
+          expect(article).toEqual(output);
+        });
+    });
+  });
+
+  describe("Error Handling", () => {
+    test("If Article ID is malformed, present an error", () => {
+      const body = { inc_votes: 1 };
+      return request(app)
+        .patch("/api/articles/one")
+        .send(body)
+        .expect(400)
+        .then(({ body }) => {
+          const { msg } = body;
+          expect(msg).toEqual("Invalid ID");
+        });
+    });
+    test("If article does not exist, an error should be presented", () => {
+      const body = { inc_votes: 1 };
+      return request(app)
+        .patch("/api/articles/1986")
+        .send(body)
+        .expect(404)
+        .then(({ body }) => {
+          const { msg } = body;
+          expect(msg).toEqual("Article Not Found");
+        });
+    });
+
+    test("If inc_votes is malformed return an error", () => {
+      const body = { inc_votes: "one" };
+      return request(app)
+        .patch("/api/articles/1")
+        .send(body)
+        .expect(400)
+        .then(({ body }) => {
+          const { msg } = body;
+          expect(msg).toEqual("Malformed Body");
+        });
+    });
+
+    test("If inc_votes is not present, return an error", () => {
+      const body = {};
+      return request(app)
+        .patch("/api/articles/1")
+        .send(body)
+        .expect(400)
+        .then(({ body }) => {
+          const { msg } = body;
+          expect(msg).toEqual("Malformed Body");
+        });
+    });
```

Comment:
Your error tests above this one are all brilliant!
This last test is reducing the ability of your API to expand in future. If one day we want to be able to PATCH _other_ properties such as the _title_ or _body_, we might not want to always include _inc_votes_ in the request body. This would require you coming back to change this test to prevent failure. 

Instead, if the _inc_votes_ property is missing, your API could return a `200` with the article _unchanged_